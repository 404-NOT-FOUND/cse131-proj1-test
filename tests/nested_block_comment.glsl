/* hey /* hey */

/* hey
 * /* hey
 * 1.0f
 */

/* 	/* */ hey */
