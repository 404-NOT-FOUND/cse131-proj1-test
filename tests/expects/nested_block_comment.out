hey          line 8 cols 15-17 is T_Identifier 
*            line 8 cols 19-19 is T_Star 
/            line 8 cols 20-20 is T_Slash 
